templates/unicorn.rb

```diff
@@ -1,30 +0,0 @@
-# https://devcenter.heroku.com/articles/rails-unicorn
-
-worker_processes (ENV["UNICORN_WORKERS"] || 3).to_i
-timeout (ENV["UNICORN_TIMEOUT"] || 15).to_i
```

Comment:
Have you addressed the exception issue detailed in https://github.com/heroku/rack-timeout/issues/76?